modules/virt-about-kubemacpool.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_vm_networking/virt-using-mac-address-pool-for-vms.adoc
+
+[id="virt-about-kubemacpool_{context}"]
+= About KubeMacPool
+
+If the KubeMacPool component is enabled for the namespace, virtual machine NICs are allocated MAC addresses from a MAC address pool.
```

Comment:
This sentence seems a little ambiguous. Suggest "If you enable the KubeMacPool component for a namespace, virtual machine NICs in that namespace are allocated..."